MonacoEditorComponent/Monaco/Languages/CompletionList.cs

```diff
@@ -11,22 +11,12 @@ namespace Monaco.Languages
     /// <summary>
     /// https://microsoft.github.io/monaco-editor/api/interfaces/monaco.languages.completionlist.html
     /// </summary>
-    public sealed class CompletionList : IEnumerable<CompletionItem> // TODO: Could I somehow also just make this a list?  Investigate Json Serialization Converter helper for that?
+    public sealed class CompletionList
```

Comment:
Reason for removing the enumerable interface?

Reply:
Yes, with that IEnumerable interface, NewtonSoft.Json serializes it as a JSON array of `CompletionItem` objects where Monaco expects a `CompletionList` object with properties `suggestions` and `incomplete`.

Without the interface inheritance the serialization works ok.

Reply:
Ah, I didn't think about the serialization aspect from that.  I should try and figure out some tests in this area, as I was having problems in the past, I wonder if it was related to this maybe.